tests/aws/services/cloudformation/resources/test_opensearch.py

```diff
@@ -35,7 +36,15 @@ def test_domain(deploy_cfn_template, aws_client, snapshot):
     result = deploy_cfn_template(template_path=template_path)
     domain_endpoint = result.outputs["SearchDomainEndpoint"]
     assert domain_endpoint
+    domain_arn = result.outputs["SearchDomainArn"]
+    assert domain_arn
```

Comment:
worth snapshotting the `result.outputs`?

Reply:
don't think so :) there would only be placeholders/replacements in the snapshot. 
We test the arn anyhow when retrieving the tags, and the domain-name for the describe. Those two responses are snapshotted.